spec/javascripts/Stats/stats_response_codes.spec.js

```diff
@@ -31,7 +31,7 @@ describe('StatsResponseCodeSource', () => {
 
 describe('StatsResponseCodeChart', () => {
   beforeEach(() => {
-    fixture.set(`<div id="chart"></div><div id="no-data">No data 'ere mate!</div>`)
+    fixture.set('<div id="chart"></div><div id="no-data">No data \'ere mate!</div>')
```

Comment:
It's ok to use template strings

Reply:
But it really isn't:
![screen shot 2018-12-19 at 09 17 15](https://user-images.githubusercontent.com/11672286/50207517-fac4d400-036e-11e9-892e-9f0d61b731c2.png)
It's not [standard.js](https://standardjs.com/rules.html#rules) compliant (second rule).

Reply:
Ok, so we are allowing only back ticks on substitution.